### DISCUSSION TITLE
about multipleinterface testcase

### DISCUSSION THREAD

**Discussion by 03psb** [ANLAB-KAIST/KENSv3#225] [Project Q&A 2. Transport Layer (TCP)] [answered]
![image](https://github.com/ANLAB-KAIST/KENSv3/assets/98043513/6b976f78-df1f-4162-a879-a05554ab2ffb)
I found in multipleinterface1 testcase of tcp handshake test, it tries to bound twice to a same fd. I'm confused since in test bind there exists testcase that exactly bans this behavior. Can you explain more about this testcase?

**Comment by Pusnow** [comment, marked as answer]
Their `pid`s would differ.

In POSIX systems like Linux, each process has its own file descriptor space.
It is similar in KENS, you should use `pid` to distinguish processes and manage file descriptor  separately.
